cmd/migrate/main.go

```diff
@@ -41,6 +43,11 @@ func main() {
 	if err := goose.Run(command, db, *dir); err != nil {
 		log.Fatalf("goose %v: %v", command, err)
 	}
+	config.DB,err_=gorm.Open("mysql",config.DbURL(config.BuildConfig()))
```

Comment:
remove this from migration file.

Reply:
shall i put it in main.go of cmd folder ?